### DISCUSSION TITLE
【求助】我的世界服务器经常会出现报错，求一位大佬前来帮助

### DISCUSSION THREAD

**Discussion by H-luoyan** [share121/inter-knot#2684] [其他]
如果需要完整的日志文件，可以加QQ3117669520，QQ群916641081进行联系，也可以直接进入服务器111.161.122.206:49170进行解决，感谢各位的帮助
![8e7cb3674f5edcd99bb21632de2d8b7](https://github.com/user-attachments/assets/3e430686-60f7-4603-86e1-ab6a54ae646e)

**Comment by alanyagger** [comment]
看spigot的报错信息推测是服务器负载太高崩溃了 可以看看有没有卡服机或者刷实体数量过多 或者提升一下服务器配置

**Reply by H-luoyan** [reply]
具体是内存过高还是HDD磁盘占用过高导致读取数据失败？

**Reply by alanyagger** [reply]
> 具体是内存过高还是HDD磁盘占用过高导致读取数据失败？

我水平比较一般 🫥🫥只看这张图暂时看不出来是内存瓶颈还是硬盘IO😢

**Comment by richi-shek** [comment]
你可以用Timing来查看服务端的性能分析报告

1. 在服务器内或者直接在后台内输入指令 —— timings on 开启统计
2. 等待至少15分钟(很卡顿的话可以等待5分钟)
3. 在等待足够的时间后，输入指令 —— timings paste
4. 打开它给你的网址，查看性能分析报告

如果你看不懂性能分析报告，可以看
https://shockbyte.com/billing/knowledgebase/220/How-to-Analyze-a-Timings-v2-Report.html

最后，不需要使用的时候请使用 timings off 指令关闭性能统计，否则会带来不必要的性能损耗。

**Reply by richi-shek** [reply]
另外，你可以优化spigot.yml或bukkit.yml
你可以看
https://www.minebbs.com/threads/minecraft-server-optimization-guide.27098/

**Reply by richi-shek** [reply]
最后，我建议你有问题可以先问百度或谷歌，查"我的世界玩服务器卡怎么办"，有一大堆教程

**Reply by H-luoyan** [reply]
感谢您的帮助，之前我也去搜索过，怀疑可能是插件问题，但是没有解决，在其他平台上没有流量，如果真的是硬件问题，我也已经采购了SSD

**Reply by richi-shek** [reply]
@H-luoyan log中"The server has stopped responding"指的是服务器太卡了，卡到"Stopping server"
1. 你只装BlockLocker和SkinsRestorer两个插件，这两个插件不占太多的cpu，而且你插件没有报错，不是插件的问题
2. 有SSD，而且你服务器只有5个玩家，存储的区块不多，不是硬盘IO的问题

所以应该是刷实体数量过多的关系，建议优化server.properties、spigot.yml和bukkit.yml，减轻服务器负担或换更好的CPU

**Reply by H-luoyan** [reply]
目前服务器用的是HDD，有一次我在远距离TP的时候报错关服，所以我怀疑是磁盘的问题

**Reply by richi-shek** [reply]
log的报错指的是服务器太卡了，卡到关服，有几种可能：
1. 磁盘的问题，远距离TP需要读取区块文件，磁盘有问题的话可能读取过慢，建议换SSD
2. CPU的问题，远距离TP需要读取区块文件，如果区块文件不存在，服务器需要生成区块，加上服务器其他负担(实体数量过多、磁盘的问题)，CPU负担太重
3. 记忆体(ram)的问题，服务器调用的记忆体过少，可能导致卡顿

你可以用Timing来查看服务端的性能分析报告，看看什么导致卡顿

**Comment by NekoWise** [comment]
有没有一种可能，你现在是有求于人，为何不直接放出完整的日志，而是让受委托人去主动加你呢

**Reply by H-luoyan** [reply]
完整的日志有8万字符，就算是截图也要十多张，在QQ我可以直接把原文件发过去

**Reply by H-luoyan** [reply]
我开始不知道绳网上可以发除了文字和图片以外的东西，现在我试了可以，已经发在下面了

**Comment by H-luoyan** [comment]
[latest.log](https://github.com/user-attachments/files/16647604/latest.log)

**Comment by 1luik** [comment]
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/builddata/browse/mappings/bukkit-1.21.1-cl.csrg
你的报错大部分都是没有任何作用的，bukkit只有class原版的混淆函数名字都是一样的不一样的只有desc
只能用专业软件拦截或者改他的报错程序

**Reply by 1luik** [reply]
我可以勉强算大佬吧我主要是写mod的
